app/services/core_data_connector/import/instances.rb

```diff
@@ -0,0 +1,116 @@
+module CoreDataConnector
+  module Import
+    class Instances < Base
+      def cleanup
+        super
+
+        execute <<-SQL.squish
```

Comment:
Since these two SQL statements don't rely on each other, we can probably write this as two separate SQL statements.